src/Summoner/Source.hs

```diff
@@ -0,0 +1,55 @@
+{-# LANGUAGE ViewPatterns #-}
+
+module Summoner.Source
+       ( Source (..)
+       , sourceT
+       , fetchSource
+       ) where
+
+import Relude
+
+import Control.Arrow ((>>>))
```

Comment:
What do you think if `relude` exports `>>>` and `<<<`? 🤔 
Looks like this operators are very common. And it has unambiguous name.